features/convergedcloud/pkg.include

```diff
@@ -1,14 +1,15 @@
 apparmor
-open-vm-tools
+arptables
 containerd
+curl
 docker.io
+ebtables
 ethtool
-socat
-curl
-logrotate
+gnupg
 jq
-nfs-common
+logrotate
 netcat-openbsd
-ebtables
-arptables
+nfs-common
```

Comment:
Less is more. NFS should only be installed by platforms requiring it instead of being preselected.

Reply:
This list was only sorted for better readability, so I'd do this in #17